timescaledb/how-to-guides/skipscan.md

```diff
@@ -0,0 +1,83 @@
+# SkipScan
+SkipScan is a feature intended to vastly improve query times for `DISTINCT`
+queries. Skip Scan works on PostgreSQL tables, TimescaleDB hypertables, and
+TimescaleDB distributed hypertables. SkipScan is included in TimescaleDB 2.2.1
+and above.
+
+## About SkipScan
+When you query your database to find the most recent value of an item, you
+usually use a `DISTINCT` query. For example, if you wanted to find the latest
+stock or cryptocurrency price for each of your investments, or you have graphs
+and alarms that repeatedly query the most recent values for every device or
+service.
+
+As your tables get larger, `DISTINCT` queries tend to get slower. This is
+because PostgreSQL does not currently have a good mechanism for pulling a list
+of unique values from an ordered index. Even when you have an index that matches
+the exact order and columns for these kinds of queries, PostgreSQL scans the
+entire index to find all unique values. As a table grows, this operation keeps
+getting slower.
+
+SkipScan allows queries to incrementally jump from one ordered value to the next
+without reading all of the rows in between. Without support for this feature,
+the database engine has to scan the entire ordered index and then de-duplicate
+at the end, which is a much slower process.
+
+<highlight type="note">
+PostgreSQL has plans to implement a native feature like SkipScan, but it is
+unlikely to be included until at least PostgreSQL&nbsp;15. This section
+documents TimescaleDB SkipScan, which is not a native PostgreSQL feature.
+</highlight>
+
+SkipScan is an optimization for queries of the form `SELECT DISTINCT ON
+<column>`. Conceptually, SkipScan is a regular IndexScan that skips across an
+index looking for the next value that is greater than the current value.
+
+When you use a query with SkipScan, the `EXPLAIN` output includes a new node
+that can quickly return distinct items from a properly ordered index. With an
+IndexOnly scan, you have to scan the entire index, but SkipScan incrementally
+searches for each successive item in the ordered index. As it locates one item,
+the SkipScan node quickly restarts the search for the next item. This is a much
+more efficient way of finding distinct items in an ordered index.
+
+For benchmarking information on how SkipScan compares to regular `DISTINCT`
+queries, see our [SkipScan blog post][blog-skipscan].
+
+## Use SkipScan queries
```

Comment:
Might help to make it clearer that the query planner chooses SkipScan, it's not something the user has to turn on, but these choices make it possible for the query planner to choose SkipScan

Otherwise, the user reader be wondering where they turn it on